### ISSUE TITLE
labels for columns in benchmark

### ISSUE THREAD

**Issue by amueller** [pvn25/ML-Data-Prep-Zoo#1]
hey! I was trying to run the benchmark, but it's not clear to me where the ground truth labels are stored.
I assume ``meta_data.csv`` maps the column id to the csv file it came from, right?

Thanks!

**Comment by vraj-ucsd** [comment]
Hi,

The `y_act` column in the `data_train` and `data_test` files denote the ground truth label. The label vocabulary is coded as follows:

Numeric : 0
Categorical: 1
Datetime:2
Sentence:3
URL: 4
Numbers: 5
List: 6
Not-Generalizable: 7
Custom Object (or Context-Specific): 8

I've updated the readme to reflect this. I'm really sorry for this confusion. 

Yes,  the `meta_data.csv` maps every column to the source CSV file. So, a join on `Record_id` in `data_train` and `meta_data` would give the source file of that column. The raw CSV files can be downloaded from [here](https://drive.google.com/file/d/1ZPZY2wvDvsmnpQBABLz9ZyZRGvkEmo7B/view). 

Thank you for pointing this out.

**Comment by amueller** [comment]
thank you for the quick reply, I'll give it a go next week!

**Comment by amueller** [comment]
Took a bit longer, results on the training set are here:
```
                   precision    recall  f1-score   support

          Numeric       0.78      0.86      0.82      2909
      Categorical       0.51      0.64      0.57      1854
         Datetime       0.00      0.00      0.00       549
         Sentence       0.17      0.69      0.27       293
              URL       0.00      0.00      0.00       120
          Numbers       0.00      0.00      0.00       469
             List       0.00      0.00      0.00       188
Not-Generalizable       0.44      0.60      0.51       848
    Custom Object       0.00      0.00      0.00       706

         accuracy                           0.56      7936
        macro avg       0.21      0.31      0.24      7936
     weighted avg       0.46      0.56      0.50      7936
```
![image](https://user-images.githubusercontent.com/449558/103029744-4dfa3000-450f-11eb-8939-2c10350baad8.png)

Some categories are not produced by dabl right now, but I'm also a bit confused by some of the mistakes, see #4
